experimental/rel_component/configs/rel_tok2vec.cfg

```diff
@@ -0,0 +1,114 @@
+[paths]
+train = null
+dev = null
+raw = null
+init_tok2vec = null
+
+[system]
+seed = 342
+gpu_allocator = null
+
+[nlp]
+lang = "en"
+pipeline = ["tok2vec", "relation_extractor"]
+disabled = []
+before_creation = null
+after_creation = null
+after_pipeline_creation = null
+tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}
+
+[components]
+
+[components.tok2vec]
+factory = "tok2vec"
+
+[components.tok2vec.model]
+@architectures = "spacy.HashEmbedCNN.v1"
+pretrained_vectors = null
+width = 96
+depth = 2
+embed_size = 300
```

Comment:
Hm, this `embed_size` setting is named that way for backwards compatibility, but it's really the number of _rows_ in the hash embed:

```
embed_size (int): The number of rows in the hash embedding tables. This can be surprisingly small, due to the use of the hash embeddings. Recommended values are between 2000 and 10000.
```

Try a higher value, if you haven't yet. I'd be surprised if 300 is enough.

Reply:
This `embed_size = 300` crept in from the NEL configuration. I fixed it both here and there (and will fix it in the spaCy default config too)